lib/src/driver.dart

```diff
@@ -388,6 +406,19 @@ class Driver {
     return null;
   }
 
+  Map<String, List<fileSystem.Folder>> _getPackageMap(Packages packages) {
+    if (packages == null) {
+      return null;
+    }
+
+    Map<String, List<fileSystem.Folder>> folderMap =
+        new Map<String, List<fileSystem.Folder>>();
+    packages.asMap().forEach((String path, Uri uri) {
+      folderMap[path] = [PhysicalResourceProvider.INSTANCE.getFolder(uri.path)];
```

Comment:
I misspoke in my previous comment: I meant to say `uri.toFilePath(windows: Platform.isWindows)`.

However, on further reflection, I believe it is even better to do `fromUri(uri)` (fromUri() comes from the `path` package).  This is what is done in the analysis server.